### DISCUSSION TITLE
把 class 換成 collection

### DISCUSSION THREAD

**Discussion by dannypsnl** [FizzyElt/functional-programming#94] [Q&A] [answered]
set class 論題當然是正確的，除了沒有設定 class 的具體來源之前我們沒有辦法討論它的內容。而 collection 這個詞不具備任何前置條件，很容易用到更多地方不會有問題。

**Comment by FizzyElt** [comment, marked as answer]
這我覺得OK，class 還有可能因個人背景理解上有所落差
